metrics/web/lib/common/presentation/button/theme/style/metrics_button_style.dart

```diff
@@ -18,7 +18,7 @@ class MetricsButtonStyle {
   const MetricsButtonStyle({
     this.color = Colors.blue,
     this.hoverColor = Colors.black12,
+    this.elevation = 0.0,
```

Comment:
I think our documentation approach is to specify all default values? In this case we seem to lack them.